pymisp/mispevent.py

```diff
@@ -1559,7 +1559,8 @@ def __repr__(self) -> str:
 class MISPEvent(AnalystDataBehaviorMixin):
 
     _fields_for_feed: set[str] = {'uuid', 'info', 'threat_level_id', 'analysis', 'timestamp',
-                                  'publish_timestamp', 'published', 'date', 'extends_uuid'}
+                                  'publish_timestamp', 'published', 'date', 'extends_uuid',
+                                  'protected'}
```

Comment:
Just a note: we need to add in `def _set_default` an entry `self.protected = False`, to keep consistency with the other required fields.

I'm assuming protected set to false par default is the expected value.